src/smolvm/facade.py

```
@@ -925,7 +939,15 @@ def start(
                     "the rootfs at build time.",
                     {"vm_id": self._vm_id},
                 )
-            self.wait_for_ssh(timeout=boot_timeout, on_progress=on_progress)
+            try:
+                self.wait_for_ssh(timeout=boot_timeout, on_progress=on_progress)
+                if self._telemetry:
+                    self._telemetry.record_boot_ready()
+            except OperationTimeoutError as e:
+                if self._telemetry:
+                    self._telemetry.record_boot_failure(f"SSH timeout: {str(e)}")
```

Ruff (0.15.12)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)